lib/config/validation.ts

```diff
@@ -199,10 +199,12 @@ export async function validateConfig(
       if (
         !isPreset &&
         optionParents[key] &&
-        optionParents[key] !== parentName
+        !optionParents[key]?.includes(parentName)
```

Comment:
```suggestion
        !optionParents[key].includes(parentName)
```
validated right before